tests/test_form.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''Tests for form validation.'''
+
+import unittest
+from webcompat import form
+
+
+class TestForm(unittest.TestCase):
+
+    def test_normalize_url(self):
+
+        r = form.normalize_url('example.com')
```

Comment:
Can we add a test for the instances where nothing should happen, i.e., `http://example.com` and `https://example.com`? (Right now those will fail, will comment below in a sec)